mlir-tensorrt/test/Dialect/Plan/refine-types.mlir

```diff
@@ -126,8 +126,24 @@ func.func @tensorrt_opaque_plugin_no_cast() -> tensor<?xf32> {
   return %3 : tensor<?xf32>
 }
 // CHECK-LABEL: func.func @tensorrt_opaque_plugin_no_cast
-// CHECK-SAME: () -> tensor<?xf32>
-// CHECK: %[[v1:.*]] = stablehlo.dynamic_reshape %{{.*}}, %{{.*}} : (tensor<64xf32>, tensor<1xi32>) -> tensor<64xf32>
-// CHECK: %[[v2:.*]] = tensorrt.opaque_plugin
-// CHECK-SAME: (%[[v1]]) : (tensor<64xf32>) -> tensor<?xf32>
-// CHECK: return %{{.*}} : tensor<?xf32>
\ No newline at end of file
+// CHECK-SAME: () -> tensor<1xf32>
+// CHECK: %[[cst:.*]] = stablehlo.constant dense<1.000000e+00> : tensor<64xf32>
+// CHECK: %[[v0:.*]] = tensorrt.opaque_plugin
+// CHECK: return %{{.*}} : tensor<1xf32>
+
+// -----
+
+func.func @tensorrt_resize_no_cast() -> tensor<1x1x8x8xf32> {
+  %cst = stablehlo.constant dense<1.000000e+00> : tensor<1x1x4x4xf32>
+  %c = stablehlo.constant dense<[1, 1, 8, 8]> : tensor<4xi32>
```

Comment:
Remove this test if you don't keep the original pattern. A test is needed for the pattern `TensorRTRefineTypeFromWithShapeGeneric`, however.

Reply:
Replaced the test with `refine_tensorrt_resize_with_shape`.